uaclient/messages.py

```diff
@@ -102,6 +102,7 @@ class TxtColor:
 LIVEPATCH_LTS_REBOOT_REQUIRED = (
     "Livepatch support requires a system reboot across LTS upgrade."
 )
+FIPS_REBOOT_REQUIRED_MSG = "Reboot to FIPS kernel required"
```

Comment:
I guess `FIPS_REBOOT_REQUIRED` should be enough

Reply:
It should, but we have a `NamedMessage` with this name already and I don't want to further expand on this at this moment

Reply:
makes all sense